README.md

```diff
@@ -1,70 +1,71 @@
-# Getting Started with Create React App
+# Bike-Shop-Frontend-App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A Bike Shop App developed using React for Front-End and Ruby on Rails for Back-End.
 
-## Available Scripts
+![image](https://user-images.githubusercontent.com/91697183/192991047-32537abd-f1f1-4e7f-97fa-a9dbc1842304.png)
 
-In the project directory, you can run:
+This is our Final React-Rails Capstone project in the Microverse curriculum.
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- REACT
+- CSS
+- API
+- JSX
+- REACT-ROUTER-DOM
+ 
+ ## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Frontend Hosted Link](https://bike-shop-cmlx.netlify.app/)
 
-### `npm test`
+## Getting Started
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+To get a local copy up and running:
 
-### `npm run build`
+1. Clone this repository or download the Zip folder:
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+**``git clone https://github.com/ger619/bike-shop-frontend``**
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+1. Navigate to the location of the folder in your machine:
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+**``you@your-Pc-name:~$ cd <folder>``**
```

Comment:
- [ ] On your readme, there are missing steps to run your app.  Your users will need to run `npm install` and then `npm start`.  Please, kindly consider documenting this 2 missing steps 👍